source/common/json/config_schemas.cc

```diff
@@ -954,6 +954,26 @@ const std::string Json::Schema::TOP_LEVEL_CONFIG_SCHEMA(R"EOF(
         "required" : ["type", "config"],
         "additionalProperties" : false
       },
+      "zipkin_driver" : {
+        "type" : "object",
+        "properties" : {
+          "type" : {
+            "type" : "string",
+            "enum" : ["zipkin"]
```

Comment:
My original code had both "lightstep" and "zipkin" as valid values of the same enum, and I had made  `access_token_file` an optional parameter, as it has no use for Zipkin. However, under PR #700 (the old PR replaced by this one), @RomanDzhabarov suggested that the following: "this should be done in a way that there are separate schemas for different driver types, otherwise schema will not have much value."